charters/charter-template.md

```diff
@@ -0,0 +1,30 @@
+
+# Project Name
+
+
+## Description
+
+## Objective
+
+## In-scope
```

Comment:
Re. module alignment with HCA Governance, one option is to add a `Scientific guardrails` subsection here. A `Scientific guardrails` section could describe contact people and processes that governing and science orgs. hold with DCP modules to make sure that developments in the module suit the needs of the scientific community. This section could also mandate reviews by the governing and science orgs. of developments in the module that require oversight.

/cc @TimothyTickle @brianraymor

Reply:
@mweiden - Reviewed your proposal with @TimothyTickle. Including a `Scientific guardrails` section in the template and requiring an explicit review by the Science PM in the process addresses the _alignment with HCA Governance_ concerns. I'll add in the next update.